### DISCUSSION TITLE
Trying to clone, asking for authentication!!

### DISCUSSION THREAD

**Discussion by maharajg** [prasanthrangan/hyprdots#162] [Q&A]
When trying to clone, im being asked to authenticate? only happening for this.

**Comment by prasanthrangan** [comment]
ohh it should not, are you cloning the correct repo? please double check the link.

**Reply by Gedankenn** [reply]
if you try to clone it with ssh it will ask for password. you must clone it with http. are you guys cloning it with ssh? :octopus:

**Reply by maharajg** [reply]
this is the command i'm trying, git clone https://github.com/prasanthrangan/hyprdots ~/Hyprdots
